scripts/update-apk.sh

```diff
@@ -53,4 +62,4 @@ if [ "$TRAVIS_BRANCH" != "$PUBLISH_BRANCH" ]; then
 fi
 
 gem install fastlane
-fastlane supply --apk test-app-release.apk --track alpha --json_key ../scripts/fastlane.json --package_name $PACKAGE_NAME
+fastlane supply --apk eventyay-organizer-app-release.apk --track alpha --json_key ../scripts/fastlane.json --package_name $PACKAGE_NAME
```

Comment:
Wrong name here

Reply:
So it should be eventyay-organizer-master right?

Reply:
Yes

Reply:
You can leave it to be app-release, it will work as well